aloha-core/src/test/scala/com/eharmony/aloha/models/multilabel/MultilabelModelTest.scala

```diff
@@ -47,124 +63,336 @@ class MultilabelModelTest extends ModelSerializationTestHelper {
+    result match {
+      case Success(_) => fail()
+      case Failure(ex) => assertEquals(exception, ex)
+    }
   }
 }
 
 object MultilabelModelTest {
-  // TODO: Use this label type and Auditor.
-
+  // Types
   private type Label = String
   private val Auditor = RootedTreeAuditor.noUpperBound[Map[Label, Double]]()
 
-// TODO: Access information returned in audited value by using the following functions:
-  //    val aud: RootedTree[Any, Map[Label, Double]] = ???
-  //    aud.modelId           // : ModelIdentity
-  //    aud.value             // : Option[Map[Label, Double]]  // Should be missing on failure.
-  //    aud.missingVarNames   // : Set[String]
-  //    aud.errorMsgs         // : Seq[String]
-  //    aud.prob              // : Option[Float]  (Shouldn't need this)
+  case class ConstantPredictor[K](prediction: Double = 0d) extends SparseMultiLabelPredictor[K] {
+    override def apply(featuresUnused: SparseFeatures,
+      labels: Labels[K],
+      indicesUnused: LabelIndices,
+      ldfUnused: SparseLabelDepFeatures): Try[Map[K, Double]] =
+        Try(labels.map(_ -> prediction).toMap)
+  }
+
+  case class Lazy[A](value: A) extends (() => A) {
+    override def apply(): A = value
+  }
+
+  // Common input
+  val labelsInTrainingSet: sci.IndexedSeq[Label] = sci.IndexedSeq[Label]("a", "b", "c")
+  val labelsInTrainingSetToIndex: Map[Label, Int] =labelsInTrainingSet.zipWithIndex.toMap
+  val labelsNotInTrainingSet: Seq[Label] = Seq("a", "b")
+  val baseErrorMessage: Seq[String] = Stream.continually("Label not in training labels: ")
+  val errorMessages: Seq[String] = baseErrorMessage.zip(labelsNotInTrainingSet).map {
+    case(msg, label) => s"$msg$label"
+  }
+  val auditor: RootedTreeAuditor[Any, Map[Label, Double]] =
+    RootedTreeAuditor[Any, Map[Label, Double]]()
+
+  // Feature functions
+  val EmptyIndicatorFn: GenAggFunc[Map[String, String], Iterable[(String, Double)]] =
+    GenFunc0("", _ => Iterable())
+  val featureFunctions = Vector(EmptyIndicatorFn)
+
+  // Models
+  val modelNoFeatures = MultilabelModel(
+    modelId             = ModelId(),
+    featureNames        = sci.IndexedSeq(),
+    featureFunctions    = sci.IndexedSeq[GenAggFunc[Int, Sparse]](),
+    labelsInTrainingSet = sci.IndexedSeq[Label](),
+    labelsOfInterest    = None,
+    predictorProducer   = Lazy(ConstantPredictor[Label]()),
+    numMissingThreshold = None,
+    auditor             = Auditor
+  )
+
+  val modelWithFeatureFunctions:
+    MultilabelModel[Tree[Any], String, Vector[String], RootedTree[Any, Map[Label, Double]]] =
+    modelNoFeatures.copy(
+    featureFunctions    = sci.IndexedSeq[GenAggFunc[Vector[String], Sparse]](),
+    labelsInTrainingSet = labelsInTrainingSet,
+    labelsOfInterest    = Some(GenFunc0("", (a: Vector[String]) => a))
+  )
+
+  // LabelsAndInfo
+  val labelsAndInfoEmpty = LabelsAndInfo(
+    indices = sci.IndexedSeq.empty,
+    labels = sci.IndexedSeq.empty,
+    labelsNotInTrainingSet = Seq[Label](),
```

Comment:
Just do `Seq.empty[Label]`.  See [GenericCompanion.scala](https://github.com/scala/scala/blob/v2.11.8/src/library/scala/collection/generic/GenericCompanion.scala#L38-L46) for more info.